src/app/personside/infotabs/dyplenker.ts

```diff
@@ -0,0 +1,41 @@
+import moment from 'moment';
+import { usePaths } from '../../routes/routing';
+import { Utbetaling } from '../../../models/utbetalinger';
+import { RouteComponentProps } from 'react-router';
+import { Traad } from '../../../models/meldinger/meldinger';
+import { useRestResource } from '../../../utils/customHooks';
+import { hasData } from '../../../rest/utils/restResource';
+
+export function useInfotabsDyplenker() {
+    const paths = usePaths();
+    return {
+        utbetaling: {
+            link: (utbetaling: Utbetaling) => `${paths.utbetlainger}/${moment(utbetaling.posteringsdato).unix()}`,
```

Comment:
Ikke krise her. Men burde vi vurdere å vri oss bort fra moment?

Reply:
🤷‍♂ I dunno. Er brukt ganske tungt i appen, da er det isåfall ganske mye som må skrives om. Foreslår å lage en jira-oppgave på det i såfall

Reply:
Tenkte mer på at vi ikke introduserte mer bruk av moment om vi ønsker å gå bort fra det.
For fjerning av moment i hele kodebasen er out-of-scope for denne PRen, og bør nok være en egen jira-task ja.

Reply:
👍 Kan godt bli med på det. Hvilket bibliotek er det vi ønsker å gå over til? Og hvor stor gevinst forventer vi på en migrasjon..? 🤔